src/com/mambu/apisdk/services/UsersService.java

```diff
@@ -284,5 +287,31 @@ public Role getUserRole(String roleKey) throws MambuApiException {
 		// See MBU-9263
 		return serviceExecutor.execute(getUserRole, roleKey);
 	}
+	
+	/**
+	 * Creates a new user
+	 * 
+	 * i.e. API call POST api/users
+	 * JSON payload: { "user": { "username": "tinker", "password": "pass1234566",
+	 * "email": "ddd@mambu.com", "title": "Mrs", "firstName": "John"... } }
+	 * NOTE: for more information on users API visit 
+	 * @see <a href="https://developer.mambu.com/customer/en/portal/articles/1162284-users-api?b_id=874">Users API</a>
+	 * 
+	 * @param user
+	 *            The user to be created
+	 * @return newly created user
```

Comment:
Can add "Must not be null". May help (optionally) specifying that the user must have a role assigned.